### PR TITLE
Add unified origin trial landing page

### PR DESCRIPTION
**Pull request by anusmitaray**
Preview: https://deploy-preview-3647--developer-chrome-com.netlify.app/docs/privacy-sandbox/unified-origin-trial

### CODE REVIEW

**Comment by heyawhite** [comment]
This doc needs to be added to the table of contents on `/docs/privacy-sandbox/` and `/privacy-sandbox`

The first ToC has [handbook instructions](https://developer.chrome.com/docs/handbook/how-to/add-a-project/#configure-the-table-of-contents-for-your-project) but the tl;dr is you need to add it to: 

- Add it to `/site/_data/docs/privacy-sandbox/toc.yml`
   - Makes sense to me it would be under "Get started". Could even be first in the list, above feedback.

For `/privacy-sandbox`
- Add it to `/site/en/privacy-sandbox/index.njk` in the `intro_related` section, line 18, above FAQ (add a custom short title here).
- Add it to `/site/ja/privacy-sandbox/index.njk` in the `intro_related` section, line 12, above FAQ (don't add a title here, as it will use the Japanese language title).

File: site/en/docs/privacy-sandbox/unified-origin-trials/index.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+      <td>
+Shared Storage added to the origin trial in only M105+ Canary/Dev/Beta for now.
+	  </td>
+    </tr>
+  </tbody>
+</table>
+
+{% Details %}
+{% DetailsSummary %}
+### Previous updates
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/unified-origin-trials/index.md:105]:
```suggestion

### Previous updates
```

File: site/en/docs/privacy-sandbox/unified-origin-trials/index.md [hunk 2]
```diff
@@ -0,0 +1,370 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Privacy Sandbox relevance and measurement unified origin trials'
+subhead: >
+   The Privacy Sandbox Relevance and Measurement origin trial
+   provides a single trial allowing sites to run unified experiments
+   across Attribution Reporting, FLEDGE, Topics, Fenced Frames,
+   and Shared Storage.
```
[the remark this answers is not in the document]
Comment (heyawhite) [reply site/en/docs/privacy-sandbox/unified-origin-trials/index.md:8]: This was quite a long subhead, and as it sits below the title, it doesn't need to repeat the title.

File: site/en/docs/privacy-sandbox/unified-origin-trials/index.md [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+description: >
+   The Privacy Sandbox Relevance and Measurement origin trial
+   provides a single trial allowing sites to run unified experiments
+   across Attribution Reporting, FLEDGE, Topics, Fenced frames,
+   and Shared Storage.
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/unified-origin-trials/index.md:13]:
```suggestion
description: >
   Run unified experiments across Attribution Reporting, FLEDGE,
   Topics, Fenced Frames, and Shared Storage.
```

Comment (heyawhite) [reply site/en/docs/privacy-sandbox/unified-origin-trials/index.md:13]: Same goes for the description.

File: site/en/docs/privacy-sandbox/unified-origin-trials/index.md [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+to Experiment (I2E)</a> post.</td>
+    </tr>
+  </tbody>
+</table>
+{% endDetails %}
+
+## Sign up for and configure the origin trial {: #configure}
+
+To activate the origin trial on your site, you will need to register and embed the assigned origin trial token (a time-based string for your specific access to the origin trial). Learn more in the
+[guide to getting started with Chrome's origin trials](/blog/origin-trials/).
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/unified-origin-trials/index.md:221]:
```suggestion
To activate the origin trial on your site, you will need to register
and embed the assigned origin trial token (a time-based string
for your specific access to the origin trial). Learn more in the
[Getting started with Chrome's origin trials](/blog/origin-trials/).  
```

**Review by heyawhite — COMMENTED** [review]
Some feedback / nits. WIll approve once added to navigation

File: site/en/docs/privacy-sandbox/unified-origin-trials/index.md [hunk 5]
```diff
@@ -0,0 +1,370 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Privacy Sandbox relevance and measurement unified origin trials'
```
Comment (rowan-m) [inline site/en/docs/privacy-sandbox/unified-origin-trials/index.md:3]: This should be singular in all occurrences (including the file name). As in, there is one Privacy Sandbox Relevance and Measurement origin trial which includes multiple features—it is not multiple origin trials.

File: site/en/docs/privacy-sandbox/unified-origin-trials/index.md [hunk 6]
```diff
@@ -0,0 +1,369 @@
...
+{% Aside %}
+Register for the [Privacy Sandbox Relevance and Measurement trial](/origintrials/#/view_trial/771241436187197441). 
+{% endAside %}
+
+Origin trial tokens are granted immediately, and you can revoke or recreate them
+at any time. 
+
+For every page where you want to use the origin trial, you will need to include a trial token with that specific page's HTML or response.
+
+Use a `<meta>` tag in the top-level page's `<head>`:
```
Comment (rowan-m) [inline site/en/docs/privacy-sandbox/unified-origin-trials/index.md:231]: Could you add a line break or switch to a block-level sample for this one. In the preview, it breaks both of these over two lines which is hard to follow.

**Review by rowan-m — CHANGES_REQUESTED** [review]
A few small tweaks.

File: site/_data/docs/privacy-sandbox/toc.yml [hunk 7]
```diff
@@ -12,6 +12,7 @@
     - url: /docs/privacy-sandbox/proposal-lifecycle
 - title: i18n.docs.privacy-sandbox.start
   sections:
+	  - url: /docs/privacy-sandbox/unified-origin-trial
```
Comment (heyawhite) [inline site/_data/docs/privacy-sandbox/toc.yml:15]:
```suggestion
    - url: /docs/privacy-sandbox/unified-origin-trial  
```

File: site/en/privacy-sandbox/index.njk [hunk 8]
```diff
@@ -15,6 +15,8 @@ sections:
       description: How we collaborate with stakeholders to discuss, test, and adopt privacy-preserving technologies.
     - url: /docs/privacy-sandbox/status
   intro_related:
+    - url: /docs/privacy-sandbox/unified-origin-trial
+	  title: Privacy Sandbox unified origin trial
```
Comment (heyawhite) [inline site/en/privacy-sandbox/index.njk:19]: This should be indented as it is below
```suggestion
      title: Privacy Sandbox unified origin trial
```

File: site/en/docs/privacy-sandbox/unified-origin-trial/index.md [hunk 9]
```diff
@@ -0,0 +1,371 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Privacy Sandbox relevance and measurement unified origin trial'
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/unified-origin-trial/index.md:3]:
```suggestion
title: 'Relevance and measurement unified origin trial'
```

Strongly recommend dropping "Privacy Sandbox" as the reader is already in our docs.

DCC landing page, a long navigation.

File: site/en/docs/privacy-sandbox/unified-origin-trial/index.md [hunk 10]
```diff
@@ -0,0 +1,372 @@
...
+{% endAside %}
+
+Origin trial tokens are granted immediately, and you can revoke or recreate them
+at any time. 
+
+For every page where you want to use the origin trial, you will need to include a trial token with that specific page's HTML or response.
+
+Use a `<meta>` tag in the top-level page's `<head>`: 
+	
+`<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`.
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/unified-origin-trial/index.md:234]: A GitHub suggestion won't work, *but* this should be styled as a fenced block instead of in-line code:

https://www.markdownguide.org/extended-syntax/#fenced-code-blocks

The language is html, so after the opening three backticks, include html with no space.

File: site/en/docs/privacy-sandbox/unified-origin-trial/index.md [hunk 11]
```diff
@@ -0,0 +1,372 @@
...
+
+For every page where you want to use the origin trial, you will need to include a trial token with that specific page's HTML or response.
+
+Use a `<meta>` tag in the top-level page's `<head>`: 
+	
+`<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`. 
+
+Alternatively, include the following HTTP header in the top-level page response: 
+
+`Origin-Trial: TOKEN_GOES_HERE`
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/unified-origin-trial/index.md:238]: Same fenced code block here, with the syntax `http`

File: site/en/docs/privacy-sandbox/unified-origin-trial/index.md (hunk 9 above)
Comment (anusmitaray) [reply site/en/docs/privacy-sandbox/unified-origin-trial/index.md:3]: I don't have an opinion on this either way, but I remember @rowan-m had a reasoning behind this specific title. Can we change it?

Comment (rowan-m) [reply site/en/docs/privacy-sandbox/unified-origin-trial/index.md:3]: Yeah, this seems fair—doubling up "Privacy Sandbox" in the "Privacy Sandbox" section is a bit redundant. What I would like to retain (which we do have) is the full origin trial name in the article body, as in if I search for "Privacy Sandbox Relevance and Measurement" then this article should come up.

**Review by heyawhite — APPROVED** [review]

**Review by rowan-m — CHANGES_REQUESTED** [review]
Oops - thought I had submitted this. One last small clarification.

**Review by rowan-m — APPROVED** [review]
LGTM
